requirements.txt

```diff
@@ -1,27 +0,0 @@
-
-numpy
```

Comment:
Why remove the requirements.txt file?

Reply:
Why have two sources of truth that are copies of each other? The dependencies of dynamic-foraging should be described in the pyproject.toml. pip can (and does) use the toml, not requirements.txt to build the environment. Is it for conda? Conda just uses pip (and therefore pyproject.toml) if you do `pip install --upgrade`.

Reply:
I guess I'm not sure what the requirements.txt file was for. I thought (incorrectly) that the https://github.com/AllenNeuralDynamics/aind-library-template had both a requirements.txt and pyproject.toml. I didn't make the requirements.txt file, so I'm unsure about any unintended consequences of deleting it. 

That being said, I think we can go ahead and delete it

Reply:
I think it may have been used to set up a conda environment. I think it's redundant at the moment, and is a source of potential confusion.